packages/track-coverage/src/index.js

```diff
@@ -169,81 +203,103 @@ const CoverageTrack = ({
   const incrementSize = Math.floor(yScaleDomainMax / yTickNumber)
 
   return (
```

Comment:
Lots of indentation changes here to accommodate a new `<div>`. Changes are more obvious when viewed with `?w=1`.